.gitignore

```diff
@@ -15,6 +15,7 @@
 /eggs/
 /extends/
 /node_modules/
+/npm-shrinkwrap.json
```

Comment:
Good idea. It is recommended that this file be generated when needed. See- https://docs.npmjs.com/files/shrinkwrap.json